rpm_spec/subpackages/manageiq-appliance

```diff
@@ -73,7 +73,7 @@ if posix.access('/bin/evmserver.sh', 'x') then
   if ( pid == -1 ) then
     print ("The fork failed.")
   elseif ( pid == 0 ) then
-    posix.exec('/bin/evmserver.sh', 'update_start')
+    posix.exec('/bin/evmserver.sh', 'start')
```

Comment:
I think this will cause problems during appliance image build.  In the anaconda environment it will try to start the server, right?

Reply:
The difference is that `update_start` only starts the app if it was stopped with `update_stop`.

Reply:
so should we touch a file if we stop the service, so it will try and start it?

For regular upgrades, I don't want it to start the service either. A few times a customer has said that it started the service when they did not want it to start after upgrading their rpms